src/modules/hardwareWallet/store/actions/actionTypes.js

```diff
@@ -5,6 +5,7 @@ const actionTypes = {
   setHWDevices: `HW_SET_DEVICES`,
   addHWDevice: `HW_ADD_DEVICE`,
   removeHWDevice: `HW_REMOVE_DEVICE`,
+  updateHWAccount: `HW_UPDATE_HW_ACCOUNT`,
```

Comment:
```suggestion
  updateHWAccount: `HW_UPDATE_ACCOUNT`,
```

Reply:
or rename others to `UPDATE_HW_ACCOUNT, ADD_HW_ACCOUNT` etc..

Reply:
[Fixed](https://github.com/LiskHQ/lisk-desktop/pull/4912/commits/b0545ae4ba810bb274c748cdac956dfb254eeed8)